server/src/schemas/usersSchema.ts

```diff
@@ -0,0 +1,73 @@
+import { gql } from "apollo-server-express";
+
+export const UsersTypeDefs = gql`
+  type User {
+    id: ID!
+    first_name: String!
+    last_name: String!
+    email: String!
+    type: Type!
+    privilege: Privilege!
+    registration_date: Date!
+    balance: Int!
+    hold: Boolean!
+    holds: [Hold]
+    training_modules: [TrainingModule]
+    year: Int
+    college: String
+    major: String
+    description: String
+  }
+
+  enum Type{
+    STUDENT
+    FACULTY
+    EMPLOYEE
```

Comment:
Is there a benefit to having a distinction between `FACULTY` and `EMPLOYEE`? I'm thinking an `isStudent` boolean might be simpler.